components/AvalancheForecastZoneMap.tsx

```diff
@@ -82,21 +81,39 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
           {isReady && centers.map(center_id => <AvalancheCenterForecastZonePolygons key={center_id} center_id={center_id} setRegion={setRegion} date={date} />)}
         </MapView>
         <View style={styles.legend}>
-          <Text style={styles.legendTitle}>Avalanche Danger Scale</Text>
+          <View style={styles.legendHeader}>
+            <Text style={styles.legendTitle}>Danger Scale</Text>
+            <FontAwesome5 name="info-circle" size={16} color="blue" />
```

Comment:
Just to confirm I'm reading this right - as of this commit we don't do anything when someone interacts with this thing, right?

Reply:
Correct, behavior hasn't been specified yet ([figma discussion])(https://www.figma.com/file/Bai2w7x0vUakL5WRjPGRg7?node-id=94:94385#336437434)!